Analysis/backgrounds_halo.tex

```diff
@@ -35,13 +42,20 @@
   \caption{
     Left: $\eta$ distribution of the halo-like showers with $|\phi| < \pi/2$ and $|\phi| > \pi/2$.
     Middle and right: shift in the $\eta$ distribution of the halo-like showers with respect to the requirement on \sieie.
+    \PH{Please put selection. Label as data. I also don't see the
+      point of the two right plots. will read on}
   }
   \label{fig:halo_eta}
 \end{figure}
 
 For the distribution of Fig.~\ref{fig:halophi} to be a valid template for halo showers, it must be first confirmed that its shape is invariant under photon selection requirements. 
 However, further study of the \phig\ distribution of the halo showers indicates that the relative strength of the two prominent peaks in the distribution may change under the \sieie\ selection requirement.  
-To explain this phenomenon, one needs to look at the the \etag\ distribution of the shower populations near $\phig \sim 0$ and $\phig \sim \pi$, shown in the top portion of Figure~\ref{fig:halo_eta}. 
+To explain this phenomenon, one needs to look at the the \etag\
+distribution of the shower populations near $\phig \sim 0$ and $\phig
+\sim \pi$, shown in the top portion of
+Figure~\ref{fig:halo_eta}. \PH{Please add a sentence explaining the
+  effect now I understand why its in the plot. One needs to look make
+  me the reader draw the conclusion thats not a good idea here} 
```

Comment:
Dropped this section, since it basically introduces a problem to say we solved it.

Instead just saying that we do the folding to try to catch any potential effects.